snorkel/labeling/model/label_model.py

```diff
@@ -280,6 +280,10 @@ def _init_params(self) -> None:
         # Handle single values
         if isinstance(self.train_config.prec_init, (int, float)):
             self._prec_init = self.train_config.prec_init * torch.ones(self.m)
+        if isinstance(self.train_config.prec_init, np.ndarray):
+            self._prec_init = torch.from_numpy(self.train_config.prec_init)
+        if isinstance(self.train_config.prec_init, list):
+            self._prec_init = torch.from_numpy(np.array(self.train_config.prec_init))
         if self._prec_init.shape[0] != self.m:
```

Comment:
Let's add a check that `isinstance(self._prec_init, torch.Tensor)` since we have multiple conversion methods now. We can add a unit test to check that e.g. a string triggers this error message.